app/controllers/stories/articles_search_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Stories
+  class ArticlesSearchController < ApplicationController
+    before_action :set_cache_control_headers, only: :index
+
+    rescue_from ArgumentError, with: :bad_request
+
+    def index
+      @query = "...searching"
```

Comment:
I know you moved over this code but now is probably a good time to review it. Why are we assigning a static string to an instance variable? It also doesn't seem to be used in the view.

There may be a legitimate reason for this, I'd like to know myself. Same for `@article_index` which gets assigned in 3 different controller but doesn't seem to appear elsewhere in the codebase.

Reply:
Those variables bothered me as well. I could not find out why they exist but removing them does not seem to affect the search function and tests do not fail either. However, it did not seem like a good idea to remove variables without knowing how they are used so I let them be :sweat_smile: